modules/thermo-cycler/thermo-cycler-arduino/lid.cpp

```diff
@@ -300,6 +331,17 @@ void _motor_fault_callback()
 #endif
 }
 
+inline bool Lid::_bottom_switch_check()
```

Comment:
If this is defined as inline you may need to put the implementation in the header, I forget

Reply:
Oh! This doesn't give any errors but you are probably right. I merged the PR before noticing this comment. I'll put up a PR with the correction